app/app.js

```diff
@@ -1,20 +1,29 @@
 const express = require("express");
 const app = express();
-const { getAllTopics, getAllArticles } = require("../controllers/controllers");
+const {
+  getAllTopics,
+  getAllArticles,
+  getAllArticlesByID,
+  getComments,
+} = require("../controllers/controllers");
 
 app.get("/api/topics", getAllTopics);
 app.get("/api/articles", getAllArticles);
+app.get("/api/articles/:article_id", getAllArticlesByID);
```

Comment:
getAllArticlesByID suggests it'll return all articles, whereas we're wanting to get a single article; suggest renaming this function